src/ui/states/state-needs-auth-info.svelte

```diff
@@ -126,27 +126,27 @@
     flex-grow: 1;
   }
 
-  .form-container {
+  .rcb-form-container {
     display: flex;
     flex-direction: column;
     width: 100%;
     margin-top: 32px;
     margin-bottom: 16px;
   }
 
-  .form-label {
+  .rcb-form-label {
     margin-top: 8px;
     margin-bottom: 8px;
     display: block;
     font-weight: 500;
     line-height: 22px;
   }
 
-  .form-input.error input {
+  .rcb-form-input.error input {
     border-color: var(--rc-color-error);
   }
 
-  .form-error {
+  .rcb-form-error {
```

Comment:
It would be great to have some kind of linter to enforce this prefix moving forward... Would need to research options though.

Reply:
indeed! I thought the same